src/item.cpp

```diff
@@ -1943,6 +1943,8 @@ nc_color item::color_in_inventory() const
         ret = c_cyan;
     } else if(has_flag("LITCIG")) {
         ret = c_red;
+    } else if( is_disgusting_for() ) {
```

Comment:
The function has lost its argument and the name became misleading. `is_filthy()` would be better.